### DISCUSSION TITLE
excecute (code cell) & stop editing cell with one shortcut

### DISCUSSION THREAD

**Discussion by yuuuxt** [microsoft/vscode-jupyter#10173] [General] [answered]
There was discussion about markdown cell: https://github.com/microsoft/vscode-jupyter/issues/7042

Is there a way to do this for code cell, which is the default behaviour for Jupyter Notebook? 

Tried setting `Execute Cell` and `Stop editing cell` with the same shortcut, but didn't work: it just quits to command mode and doesn't execute.

**Comment by rchiodo** [comment, marked as answer]
CTRL+ENTER should do this. It runs the cell and exits command mode. It's mapped to this:

![image](https://user-images.githubusercontent.com/19672699/171287962-e864033a-4e0d-4188-89d8-9187cec8e600.png)
